src/modules/core/mail-server/mailServer.js

```diff
@@ -0,0 +1,45 @@
+const path = require("path");
```

Comment:
in src/modules/core -> create a folder email-service -> under email-service folder create a file email.service.js -> and keep the codes of this file.

Reply:
Updated